cli/parse_pdfs.py

```diff
@@ -151,21 +156,37 @@ def download_pdf(
                 }
             },
         )
-        return None
 
-    elif response.headers["Content-Type"] != "application/pdf":
-        _LOGGER.exception(
-            "Content-Type is not application/pdf.",
-            extra={
-                "props": {
-                    "document_id": parser_input.document_id,
-                    "document_url": document_url,
-                    "response_status_code": response.status_code,
-                }
-            },
-        )
         return None
+    elif response.headers["Content-Type"] != "application/pdf":
```

Comment:
Personally I recommend we leave the code as is and then alter the content-type of the data if it is wrong. @stefl14 I can go through and list the content-types if you're still waiting on creds.